llvm/test/CodeGen/AArch64/get-active-lane-mask-extract.ll

```diff
@@ -0,0 +1,90 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 4
+; RUN: llc -mattr=+sve    < %s | FileCheck %s -check-prefix CHECK-SVE
+; RUN: llc -mattr=+sve2p1 < %s | FileCheck %s -check-prefix CHECK-SVE2p1
+; RUN: llc -mattr=+sme2   < %s | FileCheck %s -check-prefix CHECK-SME2
+target triple = "aarch64-linux"
+
+; Test combining of getActiveLaneMask with a pair of extract_vector operations.
+
+define void @f0(i32 %i, i32 %n, ptr %p0, ptr %p1) #0 {
```

Comment:
nit: Could you give the functions more meaningful names, such as `@lane_mask_2x_extract_nxv16i1_i32` or something like that? Not saying my suggestion is great, but it gives more detail about the operations.

Reply:
Done